src/sagas/movieRequest.ts

```diff
@@ -0,0 +1,17 @@
+import { takeEvery, put, call } from 'redux-saga/effects';
+import { actionsEnums } from '../common-app/actions';
+import { moviesRequestActionCompleted } from '../pods/movies/actions';
+import { moviesAPI } from '../api/movies-api'
+import { MovieEntity } from '../api/model';
+
+export function* watchMovieRequest(){
+	yield takeEvery(actionsEnums.MOVIES_REQUEST, loadMovies);
+}
+
+function* loadMovies(action){
+	const movieList = yield moviesAPI.getAllMovies()
```

Comment:
We should check why in othe project we use call to handle promises

https://redux-saga.js.org/docs/api/

Just to know why we have this effect.